apps/www/components/analytics/analytics-bento.tsx

```diff
@@ -76,7 +78,104 @@ export function AnalyticsBento() {
   );
 }
 
+type Language = {
+  name: string;
+  Icon: React.FC<LangIconProps>;
+  codeBlock: string;
+};
+
+type LanguageName = "Typescript" | "Python" | "Rust" | "Golang" | "cURL";
+
+const curlCodeBlock = `curl --request GET \\
+    --url https://api.unkey.dev/v1/keys.getKey?keyId=key_123 \\
+    --header 'Authorization: Bearer <UNKEY_ROOT_KEY>'
+`;
+
+const tsCodeBlock = `import { Unkey } from "@unkey/api";
+
+const unkey = new Unkey({ rootKey: "<UNKEY_ROOT_KEY>" });
+
+const { result, error } = await unkey.keys.get({ keyId: "key_123" });
+
+if ( error ) {
+  // handle network error
+}
+
+// handle request
+`;
+
+const pythonCodeBlock = `import asyncio
+import os
+import unkey
+
+async def main() -> None:
+    client = unkey.Client("<UNKEY_ROOT_KEY>")
+    await client.start()
+    
+    result = await client.keys.get_key("key_123")
+  
+    if result.is_ok:
+        data = result.unwrap()
+        print(data.id)
+    else:
+        print(result.unwrap_err())
+    
+    await client.close()
+`;
+
+const goCodeBlock = `package main
+
+import (
+	"context"
+	"log"
+
+	unkeygo "github.com/unkeyed/unkey-go"
+	"github.com/unkeyed/unkey-go/models/operations"
+)
+
+func main() {
+	s := unkeygo.New(
+		unkeygo.WithSecurity("<UNKEY_ROOT_KEY>"),
+	)
+
+	ctx := context.Background()
+	res, err := s.Keys.GetKey(ctx, operations.GetKeyRequest{
+		KeyID: "key_123",
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+	if res.Key != nil {
+		// handle response
+	}
+}
+`;
+
+const rustCodeBlock = `use unkey::models::GetKeyRequest;
+use unkey::Client;
+
+async fn get_key() {
+    let c = Client::new("<UNKEY_ROOT_KEY>");
+    let req = GetKeyRequest::new("key_123");
+
+    match c.get_key(req).await {
+        Ok(res) => println!("{res:?}"),
+        Err(err) => eprintln!("{err:?}"),
+    }
+}
+`;
+
+const languagesList = {
+  cURL: { Icon: CurlIcon, name: 'cURL', codeBlock: curlCodeBlock, editorLanguage: "tsx" },
```

Comment:
[prism-react-renderer supports these languages out-of-box](https://github.com/FormidableLabs/prism-react-renderer/blob/c914fdea48625ba59c8022174bb3df1ed802ce4d/packages/generate-prism-languages/index.ts#L9-L23). Hence, `editorLanguage` does not support `shell` or `bash` values. We use `tsx` here, because it looks better and more consistent with our design than say `markdown`.